unity/Assets/Scripts/SimObjPhysics.cs

```diff
@@ -881,6 +876,7 @@ public class PhysicsMaterialValues {
             Debug.LogError(this.name + " is not at uniform scale! Set scale to (1, 1, 1)!!!");
         }
 #endif
+        agentManager = GameObject.Find("PhysicsSceneManager").GetComponentInChildren<AgentManager>();
```

Comment:
It seems weird for me for sim obj physics to know about agent managers. Also possibly slow for every object to have to do a find.